.env

```diff
@@ -0,0 +1,2 @@
+EMAIL="megumi@noroff.no"
```

Comment:
.env should never be tracked and should be in your `.gitignore` file. Instead you commit a `.env.example` file without real credentials.

Reply:
I see you do have this in `.gitignore` but you need to delete this file and commit the delete as well.

Reply:
> Should I still write tests, even though I cannot run them??

Reply:
> Done. Actually, I wanted to use .env to hide credentials in cypress test but I could not import it. Could you tell me how?

Reply:
There is an example repo here: https://github.com/cypress-io/cypress-example-recipes/tree/master/examples/server-communication__env-variables

The action happens here: https://github.com/cypress-io/cypress-example-recipes/blob/master/examples/server-communication__env-variables/cypress.config.js